lib/assert.js

```diff
@@ -223,11 +223,12 @@ function _metricsTest(expected, exact) {
     }
   }
 
-  if (exact && metrics.toJSON().length !== expected.length) {
+  let metricsJSON = metrics._metrics.toJSON()
```

Comment:
interesting... i'm surprised this was working for tests? perhaps this code path isn't used by any external modules currently. i assumed the failure would be in the code testing this module itself not in the actual assert helpers for external tests.